File: src/host.py
class Host:
    def __init__(self, name="Howie Mandel"):
        self.name = name
        self.current_dialogue = ""

    def set_dialogue(self, dialogue):
        self.current_dialogue = dialogue

    def get_dialogue(self):
        return self.current_dialogue

    def greet_contestant(self, contestant):
        self.set_dialogue(f"Hello {contestant.name}, welcome to Deal or No Deal!")

    def present_cases(self):
        self.set_dialogue("Here are the cases. Choose wisely!")

    def reveal_case(self, case):
        self.set_dialogue(f"The case number {case.number} contains ${case.value}!")

File: src/iphone_app/host.swift
import Foundation

class Host {
    var name: String
    var currentDialogue: String

    init(name: String, currentDialogue: String) {
        self.name = name
        self.currentDialogue = currentDialogue
    }

    func speak() {
        print("\(name): \(currentDialogue)")
    }

    func updateDialogue(newDialogue: String) {
        self.currentDialogue = newDialogue
    }
}

let howieMandel = Host(name: "Howie Mandel", currentDialogue: "Welcome to Deal or No Deal!")